pkg/config/config.go

```diff
@@ -221,6 +221,10 @@ type LintersSettings struct {
 				Recommendations []string `mapstructure:"recommendations"`
 				Reason          string   `mapstructure:"reason"`
 			} `mapstructure:"modules"`
+			Versions []map[string]struct {
```

Comment:
please, add new params to docs - currently, it's `.golangci.example.yml`

Reply:
Also, could you please document somewhere (e.g. in description of a linter) what are differences from [depguard](https://github.com/OpenPeeDeeP/depguard) linter?

Reply:
Done, also added a reminder for contributors to this in the docs

Reply:
thank you!